proto/bragi/dota2.proto

```diff
@@ -38,6 +38,46 @@ message Dota2MatchSnapshot {
   Tournament tournament  = 8;
 }
 
+message PicksAndBansSelectionsPerTeam {
+  enum State {
+    STATE_UNSPECIFIED = 0;
+    STATE_IN_PROGRESS = 1;
+    STATE_ENDED = 2;
+  }
+
+  enum TeamType {
+    TEAM_TYPE_UNSPECIFIED = 0;
+    TEAM_TYPE_RADIANT = 1;
+    TEAM_TYPE_DIRE = 2;
+  }
+
+  message HeroSelection {
+    uint32 hero_steam_id = 1;
+    string hero_key = 2;
+  }
+
+  message RadiantSelections {
+    repeated HeroSelection bans = 1;
+    repeated HeroSelection picks = 2;
+  }
+
+  message DireSelections {
+    repeated HeroSelection bans = 1;
+    repeated HeroSelection picks = 2;
+  }
+
+  State state = 1;
+  TeamType active_team = 2;
+  bool pick = 3;
```

Comment:
What does it mean?

Reply:
Its either a `pick` or a `ban`, hence using the boolean